yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Try running `make clean` and then `cd web/src && yarn` to remove your current dependencies and then regenerate the front end ones to get back in sync